ios/FishjamRNClient.podspec

```diff
@@ -3,7 +3,7 @@ require 'json'
 package = JSON.parse(File.read(File.join(__dir__, '..', 'package.json')))
 
 Pod::Spec.new do |s|
-  s.name           = 'MembraneWebRTC'
+  s.name           = 'FishjamRNClient'
```

Comment:
packages on npm uses following naming conventions:
`RNPackageName` or `react-native-package-name`

Reply:
so i would change it to RNFishjamClient